.gitignore

```diff
@@ -4,7 +4,11 @@ cxx/build/
 docs/build/
 docs/source/doxygen
 docs/source/mspass_schema/*.csv
-**/_build/
+**/build/
 **/__pycache__/
+**/dist/
+**/mspasspy.egg-info
+**/data
```

Comment:
Why adding this? Isn't it gonna ignore the `data` directory that's in our repo?

Reply:
Because after building of the project, it creates every file in the data repo and git detects this change. Maybe there is something wrong with my build.

Reply:
hmmm... that shouldn't happen. How did you build the package? I usually do ```pip3 install --user ./ -vvv```, which installs the package to `~/.local/`. Can you try that?

Reply:
I ran the command under the python-package.yml. But I tried your command, it raised an error.

pip._internal.exceptions.InstallationSubprocessError: Command errored out with exit status 1: /usr/local/bin/python3.6 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/hw/7p_zv2c16z313fq5jg7lgbmm0000gn/T/pip-req-build-9os00vmv/setup.py'"'"'; __file__='"'"'/private/var/folders/hw/7p_zv2c16z313fq5jg7lgbmm0000gn/T/pip-req-build-9os00vmv/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/hw/7p_zv2c16z313fq5jg7lgbmm0000gn/T/pip-record-51fwpu0q/install-record.txt --single-version-externally-managed --user --prefix= --compile --install-headers /Users/william/Library/Python/3.6/include/python3.6m/mspasspy Check the logs for full command output.